templates/careers/hiring-process/index.html

```diff
@@ -31,14 +31,14 @@ <h1>Canonical's hiring process</h1>
     </div>
     <div class="u-fixed-width">
       <div class="p-image-container--cinematic is-cover u-hide--small">
```

Comment:
I think it's better to use the [embedded media](https://vanillaframework.io/docs/utilities/embedded-media) utility instead here. Also not sure if it makes sense to hide this on small screens the way we did for the banner image, @mattea-turic WDYT?
```suggestion
      <div class="u-embedded-media u-hide--small">
```